scripts/windows/build-win-application.sh

```diff
@@ -199,7 +199,7 @@ function prepare_msvcr100 {
 
 function prepare_pip {
     local version=${PIP_VER:?}
-    local url=https://pypi.python.org/packages/py2.py3/p/pip/pip-${version}-py2.py3-none-any.whl
+    local url=https://pypi.python.org/packages/9c/32/004ce0852e0a127f07f358b715015763273799bd798956fa930814b60f39/pip-8.1.2-py2.py3-none-any.whl
```

Comment:
So it is not possible anymore to get the file just with the name and version?
Maybe it would still be better to not hardcode the whole url and read at least the version from the constant (like before)? To avoid updating the constants in lines 70-71 but forgetting this url.
Actually, if we already have the md5 in the constant above, we could probably have the url there as well?